csharp/AppEncryption/AppEncryption/Persistence/DynamoDbMetastoreImpl.cs

```diff
@@ -93,11 +107,25 @@ private interface IRegionStep
 
         internal bool HasKeySuffix { get; }
 
+        /// <summary>
+        /// Builder method to initialize a <see cref="DynamoDbMetastoreImpl"/> instance.
+        /// </summary>
+        ///
+        /// <param name="region">AWS region to use with the DynamoDB implementation.</param>
+        /// <returns>The current <see cref="Builder"/> instance.</returns>
         public static Builder NewBuilder(string region)
         {
             return new Builder(region);
         }
 
+        /// <summary>
+        /// Retrieve the value associated with the keyId and the time it was created.
+        /// </summary>
+        ///
+        /// <param name="keyId">The keyId to lookup.</param>
+        /// <param name="created">The created time lookup which is converted to unix time seconds.</param>
+        /// <returns>The <see cref="JObject"/> value associated with the <paramref name="keyId"/> and
+        /// <paramref name="created"/> tuple.</returns>
```

Comment:
The key difference between this and the description in the interface is that we call out that created time is converted to unix seconds. However, since we already call that in the class description, I think a simple `<inheritdoc/>` should suffice here. Same for LoadLatest and Store